JitsiConference.js

```diff
@@ -899,6 +900,32 @@ JitsiConference.prototype.onRemoteTrackAdded = function (track) {
 };
 
 /**
+ * Callback called by the Jingle plugin when 'session-answer' is received.
+ * @param {JingleSessionPC} session the Jingle session for which an answer was
+ * received.
+ * @param {jQuery} answer a jQuery selector pointing to 'jingle' IQ element
+ */
+// eslint-disable-next-line no-unused-vars
+JitsiConference.prototype.onCallAccepted = function (session, answer) {
+    // JVB conference does not care
+    logger.error("onAcceptedCall - not implemented in JVB conference");
```

Comment:
Not really, if it would be debug we would never see it. The thing is that this method should never be called unless this a P2PEnabledConference which overrides this method. We will never see this message logged unless the code is broken.